cetmix_tower_server/models/cx_tower_plan_line.py

```diff
@@ -10,6 +12,9 @@
 
 class CxTowerPlanLine(models.Model):
     _name = "cx.tower.plan.line"
+    _inherit = [
+        "cx.tower.reference.mixin",
```

Comment:
Since this mixin has been added you also need to change the `_compute_variable_ids` method. please check how to use this method in the case when a mixin is connected: e.g. [here](https://github.com/cetmix/cetmix-tower/blob/fd29f987a08f3eb9ae3f4ce902f1cc969a6f082f/cetmix_tower_server/models/cx_tower_command.py#L123)

Reply:
Thanks, I'll check it and fix

Reply:
@

> Since this mixin has been added you also need to change the `_compute_variable_ids` method. please check how to use this method in the case when a mixin is connected: e.g. [here](https://github.com/cetmix/cetmix-tower/blob/fd29f987a08f3eb9ae3f4ce902f1cc969a6f082f/cetmix_tower_server/models/cx_tower_command.py#L123)

@GabbasovDinar Could you please elaborate why this should be done?

Reply:
@ivs-cetmix we need to refactor this method because now this model inherits the methods of mixin